packages/debug/src/browser/debug-session-manager.ts

```diff
@@ -135,6 +135,14 @@ export class DebugSessionManager {
         this.debugTypeKey = this.contextKeyService.createKey<string>('debugType', undefined);
         this.inDebugModeKey = this.contextKeyService.createKey<boolean>('inDebugMode', this.inDebugMode);
         this.breakpoints.onDidChangeMarkers(uri => this.fireDidChangeBreakpoints({ uri }));
+        this.labelProvider.onDidChange(event => {
+            for (const uriString of this.breakpoints.getUris()) {
```

Comment:
Should not it be aligned with `getBreakpoints` to update breakpoints for a currently running session properly?

Reply:
As the constructor for DebugBreakpoint sets its 'uri' property to the first 'data' parameter, surely this covers it?

Reply:
ok, tested with multiple running debug sessions, seem to work, thanks